pallets/admin-utils/src/lib.rs

```diff
@@ -209,6 +238,9 @@ pub mod pallet {
             Ok(())
         }
 
+        /// The extrinsic sets the adjustment interval for a subnet.
+        /// It is only callable by the root account.
```

Comment:
We currently say in our docs that this hyperparam must not be changed. Should the doc be revised to say a subnet owner can now change this param? See https://github.com/opentensor/bittensor/blob/staging/bittensor/commands/network.py#L307 (only changeable hyperparams are in this list, should we update this list?) cc: @garrett-opentensor

Reply:
> Should the doc be revised to say a subnet owner can now change this param
The doc is correct. This is only changeable by the root account (i.e. `sudo`/`triumvirate`, etc.)

Reply:
> > Should the doc be revised to say a subnet owner can now change this param.
> 
> The doc is correct. This is only changeable by the root account (i.e. `sudo`/`triumvirate`, etc.)

I guess I should be clearer. All hyperparams that are editable by a subnet owner do already require, by default, "sudo" (it says in the key, value pairs in the above network.py loc link). My question is, the current list does not contain the "adjustment interval" hyperparam, and hence my docs also do not have it, so should I now add this "adjustment interval" to the docs, is my question. By implication, if I do add it to the docs, then I would think someone in @garrett-opentensor's team should also add it to the list in the above network.py code.

Reply:
> > > Should the doc be revised to say a subnet owner can now change this param.
> > 
> > 
> > The doc is correct. This is only changeable by the root account (i.e. `sudo`/`triumvirate`, etc.)
> 
> All hyperparams that are editable by a subnet owner do already require, by default, "sudo" 
This is not true. These params (the ones changeable by the subnet owner) are changeable by either of `sudo` or subnet owner ("`root`"). However, this specific one is only changeable by `sudo` (i.e. not `root`)

I think adding it to the docs would be good, however, note that it is not changeable by the subnet owner.